Github.js

```diff
@@ -0,0 +1,151 @@
+{
+	"translatorID": "5e385e77-2f51-41b4-a29b-908e23d5d3e8",
+	"label": "Github",
+	"creator": "Martin Fenner",
+	"target": "^https?://(www\\.)?github\\.com/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcs",
+	"lastUpdated": "2015-04-28 17:02:52"
+}
+
+/**
+	Copyright (c) 2015 Martin Fenner
+
+	This program is free software: you can redistribute it and/or
+	modify it under the terms of the GNU Affero General Public License
+	as published by the Free Software Foundation, either version 3 of
+	the License, or (at your option) any later version.
+
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+	Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public
+	License along with this program. If not, see
+	<http://www.gnu.org/licenses/>.
+*/
+
+function detectWeb(doc, url) {
+	if(url.indexOf("/search?") != -1 && getResults(doc).length) {
+		return "multiple";
+	} else if(getResult(doc)) {
+		return "journalArticle";
+	}
+}
```

Comment:
The logic should be tweaked a little bit. If the URL has `/search?` in it it should return `false` if there are no search results. Otherwise, a search page with no results that has a `meta[@property="og:description"]` tag (for whatever reason, maybe a future change) will return "journalArticle"